cmd/yoke/cmd_takeoff.go

```diff
@@ -65,6 +68,8 @@ func GetTakeoffParams(settings GlobalSettings, source io.Reader, args []string)
 	flagset.BoolVar(&params.TestRun, "test-run", false, "test-run executes the underlying wasm and outputs it to stdout but does not apply any resources to the cluster")
 	flagset.BoolVar(&params.SkipDryRun, "skip-dry-run", false, "disables running dry run to resources before applying them")
 	flagset.BoolVar(&params.ForceConflicts, "force-conflicts", false, "force apply changes on field manager conflicts")
+	flagset.BoolVar(&params.DiffOnly, "diff-only", false, "only output the diff of provided release name and wasm resources")
```

Comment:
I would update the message to not be specific to wasm resources. This is because yoke supports arbitrary input as flight results. It does not have to necessarily be resources generated from wasm (although wasm is the golden path).

Something like: `show diff between current revision and would be applied state. Does not apply anything to cluster`.

if you have better wording that's great!